examples/database_encryption/variables.tf

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "The project ID to host the cluster in"
+}
+
+variable "cluster_name_suffix" {
+  description = "A suffix to append to the default cluster name"
+  default     = ""
+}
+
+variable "region" {
+  description = "The region to host the cluster in"
+}
+
+variable "zones" {
+  type        = "list"
+  description = "The zone to host the cluster in (required if is a zonal cluster)"
+}
+
+variable "network" {
+  description = "The VPC network to host the cluster in"
+}
+
+variable "subnetwork" {
+  description = "The subnetwork to host the cluster in"
+}
+
+variable "ip_range_pods" {
+  description = "The secondary ip range to use for pods"
+}
+
+variable "ip_range_services" {
+  description = "The secondary ip range to use for pods"
+}
+
+variable "database_encryption" {
```

Comment:
Examples in this repository serve both as tangible demonstrations that users can rely on to show how functionality is utilized, and as test cases -- but primarily as demonstrations. With that in mind, it would probably be worth considering not passing this in as a variable, instead making it immediately obvious what's going on.

Reply:
I can create example but no test cases for now. Would that be sufficient?

Reply:
Adding functionality without adding tests significantly increases the failure domain of the module, and I don't think that's a risk we should be taking.

Reply:
Per conversation out of band, let's just make sure https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/issues/185 is addressed once we have Terraform 0.12 compatibility, and this can be merged.